CallWithDoxDialer/DoxDialerCaller.m

```diff
@@ -38,10 +40,15 @@ +(instancetype _Nonnull)shared {
 #pragma mark Methods
 -(void)dialPhoneNumber:(nonnull NSString *)phoneNumber {
     if (self.isDialerInstalled) {
-        NSString *urlEscapedPhoneNumber = [phoneNumber stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet URLHostAllowedCharacterSet]];
-        NSURL *launchDialerURL = [NSURL URLWithString:[NSString stringWithFormat:@"/call?targetNumber=%@", urlEscapedPhoneNumber]
-                                        relativeToURL:self.dialerSchemeURL];
-        [self openURL:launchDialerURL];
+        
+        NSURLComponents *launchDialerURLComponents = [[NSURLComponents alloc] init];
+        launchDialerURLComponents.scheme = self.dialerScheme;
+        launchDialerURLComponents.host = @"";
+        launchDialerURLComponents.path = @"/call";
+        launchDialerURLComponents.queryItems = @[
+                                                 [[NSURLQueryItem alloc] initWithName:@"targetNumber" value:phoneNumber]
```

Comment:
The library is agnostic -- it lets the Dialer app decide.

For now, the dialer app just ignores all non-digit characters.